CHANGELOG.md

```diff
@@ -28,13 +28,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `tt pack`: systemd unit parameterizing support.
 - `tt replicaset vshard`: module to manage vshard in the tarantool replicaset.
   * `tt replicaset vshard bootstrap`: command to bootstrap vshard.
+- `tt connect`: support format for Tarantool tuples for a Tarantool
+  version >= 3.2.
 
 ### Fixed
 
 - `tt clean` no longer tries to clean files multiple times.
 - Tarantool 3 config instance fails to use 108 symbols control socket on tt start.
 - Incorrect application name in case of explicit providing config path without directories.
 - Application build failure during pack with Tarantool from the current environment.
+- Sorted by name order of columns for `table` and `ttable` formats.
```

Comment:
move changelog entries to unreleased section

Reply:
Oops, updated.